AsyncDisplayKit/ASDisplayNodeExtras.mm

```diff
@@ -251,13 +251,19 @@ static inline BOOL _ASDisplayNodeIsAncestorOfDisplayNode(ASDisplayNode *possible
 
 extern UIWindow * _Nullable ASFindWindowOfLayer(CALayer *layer)
 {
+  UIView *view = ASFindViewOfLayer(layer);
```

Comment:
Doesn't this need to ascend the superlayer path if there is no corresponding view at this level, but a parent is inside of a window?